### ISSUE TITLE
alt tag missing warning for <g-image>

### ISSUE THREAD

**Issue by tomtev** [gridsome/gridsome#395]
## Summary
Missing alt text on images is bad for accessibility and gives lower lighthouse scores. We should give a warning if alt tag is missing on <g-image>. 

### Basic example
Svelte does this. Try to remove the alt tag here:
https://svelte.dev/examples#dynamic-attributes

And you get a warning:
A11y: <img> element should have an alt attribute (7:0)

### Motivation

To help people build more accessible websites.

**Comment by EtienneR** [comment]
Does it concerns @gridsome/transformer-remark too ?

`![](./img/logo.jpg)` no alt in HTML.

**Comment by perpetualgrimace** [comment]
Slightly different recommendation here. Images without alt text should get an empty alt tag. Not all images need alt text, but if no alt tag is provided then assistive tech will read out the image src URL instead. 😬

More on this: https://davidwalsh.name/accessibility-tip-empty-alt-attributes

**Comment by missmatsuko** [comment]
I agree, `<g-image>` should append a null/empty alt attribute if no value is given for `alt`. It's valid and probably nobody wants to hear the image's file name.
